wpeverest/everest-forms
# Fix - WP admin bar design issue in safari on EVF pages

### All Submissions:






### Changes proposed in this Pull Request:



Closes #178 

### How to test the changes in this Pull Request:

1. Open Everest Forms admin pages on safari.
2. Check if WP admin bar is visible or not?

### Types of changes:




### Other information:




### Changelog entry

> Fix - WPadmin bar design issue in safari on Everest Forms pages
